.github/workflows/auto-update-gh-pages.yml

```diff
@@ -0,0 +1,35 @@
+# This GitHub Action automatically updates built Doxygen files for Moltres' documentation
+# on `https://arfc.github.io/moltres/` every time a new pull request is merged into devel
+# and subsequently into master after CIVET's continuous integration tests.
+
+name: Doxygen & Github Pages Action
+
+# Activate on pushes and pull requests into master
+on:
+  push:
+    branches: [ master ]
```

Comment:
should these be `devel` and not master?

Reply:
I was actually not sure which branch to track. On one hand, our pull requests merge directly in `devel`. On the other hand, INL's integration testing tool CIVET runs more comprehensive testing on the `devel` branch before it pushes to `master`.